docs/en/connector-v2/sink/Doris.md

```diff
@@ -17,67 +30,96 @@ Version Supported
+| doris.config        | map    | yes      | -          | This option is used to support operations such as `insert`, `delete`, and `update` when automatically generate sql,and supported formats.                                                                                                                                                    |
 
-`Doris` user password
-
-### table.identifier [string]
-
-The name of `Doris` table
-
-### sink.label-prefix [string]
-
-The label prefix used by stream load imports. In the 2pc scenario, global uniqueness is required to ensure the EOS semantics of SeaTunnel.
-
-### sink.enable-2pc [bool]
-
-Whether to enable two-phase commit (2pc), the default is true, to ensure Exactly-Once semantics. For two-phase commit, please refer to [here](https://doris.apache.org/docs/dev/sql-manual/sql-reference/Data-Manipulation-Statements/Load/STREAM-LOAD).
-
-### sink.enable-delete [bool]
+#### Supported import data formats
 
-Whether to enable deletion. This option requires Doris table to enable batch delete function (0.15+ version is enabled by default), and only supports Unique model. you can get more detail at this link:
+The supported formats include CSV and JSON
 
-https://doris.apache.org/docs/dev/data-operate/update-delete/batch-delete-manual
+## Task Example
 
-### doris.config [map]
+### Simple:
 
-The parameter of the stream load `data_desc`, you can get more detail at this link:
+> This example defines a SeaTunnel synchronization task that automatically generates data through FakeSource and sends it to Doris Sink,FakeSource simulates CDC data with schema pk_id (bigint type), name (string type), and score (int type),Doris needs to create a table sink named test.e2e_table_sink and a corresponding table for it.
 
-https://doris.apache.org/docs/dev/sql-manual/sql-reference/Data-Manipulation-Statements/Load/STREAM-LOAD
+```hocon
+env {
+  execution.parallelism = 1
+  job.mode = "BATCH"
+}
 
-#### Supported import data formats
+source {
+  FakeSource {
+    schema = {
+      fields {
+        pk_id = bigint
+        name = string
+        score = int
+      }
+    }
+    rows = [
+      {
+        kind = INSERT
+        fields = [1, "A", 100]
+      },
+      {
+        kind = INSERT
+        fields = [2, "B", 100]
+      },
+      {
+        kind = INSERT
+        fields = [3, "C", 100]
+      },
+      {
+        kind = UPDATE_BEFORE
+        fields = [1, "A", 100]
+      },
+      {
+        kind = UPDATE_AFTER
+        fields = [1, "A_1", 100]
+      },
+      {
+        kind = DELETE
+        fields = [2, "B", 100]
+      }
+    ]
+  }
+}
 
-The supported formats include CSV and JSON. Default value: CSV
+sink {
+  Doris {
+    fenodes = "doris_cdc_e2e:8030"
+    username = root
+    password = ""
+    table.identifier = "test.e2e_table_sink"
+    sink.label-prefix = "test-cdc"
+    sink.enable-2pc = "false"
```

Comment:
2pc is a nice feature, we should enable it by default in our examples, and we need to add the checkpoint interval to env

Reply:
done